modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -16,36 +16,46 @@ Adjust RAM to match your latency requirements and the size and number of simulta
 
 == Minimum Software Requirements
 
-Mule requires minimum software resources:
+Mule requires the following minimum software resources:
 
-* Supported Java versions:
+* One of the following supported Java versions:
++
+** Oracle JDK 1.8.0 (JDK 1.8.0_161 recommended). 
```

Comment:
Alejandro Nosenzo just corrected the Mule runtime RNs to reflect that the recommended JDK *for the standalone Mule runtime (not Studio)* is JDK 1.8.0_172.